crates/gateway-config/src/apq.rs

```diff
@@ -0,0 +1,19 @@
+#[derive(Debug, Clone, PartialEq, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
```

Comment:
You can use `#[serde(default)]` to fill missing fields from a `AutomaticPersistedQueries::default()` instead of having to specify in twice